cms/templates/product_page.html

```diff
@@ -11,7 +11,7 @@
 {% endblock %}
 
 {% block content %}
-    <div id="wrapper">
+    <div class='product-page' id="wrapper">
```

Comment:
Nitpick: please use double quotes here

```suggestion
    <div class="product-page" id="wrapper">
```